docs/asciidoc/modules/ROOT/pages/graph-updates/uuid.adoc

```diff
@@ -100,46 +109,80 @@ Add the `uuid`:
 
 [source,cypher]
 ----
-CALL apoc.uuid.install('Person')
+CALL apoc.uuid.create('neo4j', 'Person')
```

Comment:
should we default to the "neo4j" database?
we should probably make it the last parameter then?

Reply:
Changed